src/SourceBuild/content/repo-projects/dotnet.proj

```diff
@@ -15,9 +15,14 @@
     <RepositoryReference Include="scenario-tests" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(DotNetBuildPass)' == '2'">
+  <ItemGroup Condition="'$(DotNetBuildPass)' == '2' and '$(TargetOS)' == 'windows' and '$(TargetArchitecture)' == 'x64'">
     <RepositoryReference Remove="@(RepositoryReference)" />
     <RepositoryReference Include="sdk" DotNetBuildPass="2" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(DotNetBuildPass)' == '2' and '$(TargetOS)' == 'windows' and '$(TargetArchitecture)' == 'x86'">
```

Comment:
Consider adding a small comment above these item groups mentioning what is getting built: crossdac vs bundle